src/.eslintrc.js

```diff
@@ -23,6 +23,22 @@ module.exports = {
         ]
       }
     },
+    {
+      files: ['govuk/**/*.{cjs,js,mjs}'],
```

Comment:
Not sure if this speeds things up, but I think our source code is _only_ ESM now?

My mistake originally (appreciate you've just moved it)

```suggestion
      files: ['govuk/**/*.mjs'],
```